binr/r2r/r2r.c

```diff
@@ -34,18 +34,21 @@ static void print_state(R2RState *state, ut64 prev_completed);
 static void print_log(R2RState *state, ut64 prev_completed, ut64 prev_paths_completed);
 
 static int help(bool verbose) {
-	printf ("Usage: r2r [-vh] [-j threads] [test path]\n");
+	printf ("Usage: r2r [-vh] [-j threads] [test file/dir | .test-type]\n");
```

Comment:
It seems like the special char is `@` and not `.`

Reply:
fixed